lnpbp-0002.md

```diff
@@ -73,37 +70,30 @@ Miniscript [16].
 
 ### Commitment
 
-The **committing party**, having a message `msg`
+The **committing party**, having a message `msg`:
 1. Collects all public keys instances (named **original public keys**) related to the given transaction output under this standard,
    namely:
    - a single public key for P2PK, P2PKH, P2WPK, P2WPK-in-P2SH type of outputs, `redeemScripts` or custom non-P2(W)SH
      `scriptPubkey` requiring a single signature;
    - an arbitrary public key (even with an unknown corresponding private key), if `OP_RETURN` `scriptPubkey` is used;
-   - an intermediate public key for the Taproot;
+   - an internal public key for a Taproot output;
    - all signing public keys from a `redeemScript` or custom non-P2(W)SH `scriptPubkey`, defined according to the
```

Comment:
```suggestion
   - all public keys from a `redeemScript` (for P2(W)SH and P2WSH-in-P2SH, which in case of SegWit is contained within `witnessScript`) or `scriptPubkey` (for custom bare script outputs), extracted according to the
```

Since algorithm below does not validates that the key satisfaction is required, we should probably remove this here as well

Reply:
By "key satisfaction", you mean that we don't verify that we have a valid signature for each public keys?

Reply:
By "key satisfaction", do you mean that we don't verify we have a valid signature for a public key ?